agent/engine/engine_integ_test.go

```diff
@@ -943,6 +943,9 @@ func TestSweepContainer(t *testing.T) {
 // Namely, this test verifies that Docker does emit a 'die' event after an OOM
 // event if the init dies.
 func TestInitOOMEvent(t *testing.T) {
```

Comment:
Nit: Can you also update the documentation for this function with a link to the [docker PR #4251](https://github.com/docker/docker/pull/4251)?